dask/dataframe/tests/test_rolling.py

```diff
@@ -177,5 +195,9 @@ def test_rolling_partition_size():
                     reason="rolling object not supported")
 def test_rolling_repr():
     ddf = dd.from_pandas(pd.DataFrame([10]*30), npartitions=3)
-    assert repr(ddf.rolling(4)) in ['Rolling [window=4,axis=0]',
-                                    'Rolling [axis=0,window=4]']
+    assert repr(ddf.rolling(4)) in ['Rolling [window=4,center=False,axis=0]',
+                                    'Rolling [window=4,axis=0,center=False]',
+                                    'Rolling [center=False,axis=0,window=4]',
+                                    'Rolling [center=False,window=4,axis=0]',
+                                    'Rolling [axis=0,window=4,center=False]',
+                                    'Rolling [axis=0,center=False,window=4]']
```

Comment:
At some point it will make sense to just test the following:

```python
r = repr(ddf.rolling(4))
assert 'window=4' in r
assert 'center=False' in r
assert 'axis=0' in r
assert 'Rolling' in r
```